Task03Triangles/Triangles/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿// <copyright file="Program.cs" company="Alex Brylov">
+// Copyright (c) Alex Brylov. TASK 3 - TRIANGLES
+// </copyright>
+
+namespace Triangles
+{
+    using System;
+
+    /// <summary> Contains an entry point of the application </summary>
+    internal class Program
+    {
+        /// <summary> An entry point of application </summary>
+        /// <param name="args"> Optional arguments to be passed </param>
+        private static void Main(string[] args)
+        {
+            string[] localArgs = { "First", "23", "17.5", "12" };
```

Comment:
Anton, I put "localArgs" in every task for testing purposes. At the final stage all tasks works with command line arguments provided by user at launch of application. So this line is to be removed. But on testing stage it is much more convenient to launch application by pushing Ctrl+F5 from Visual stusio instead of using Cmd.